src/sonic-yang-models/yang-models/sonic-fabric-monitor.yang

```diff
@@ -43,6 +43,19 @@ module sonic-fabric-monitor{
                     description "Consecutive polls with lesser error rate for inclusion.";
                 }
 
+                leaf monCapacityThreshWarn {
+                    type uint32;
+                    default 10;
+                    description "Percentage of up fabric links.";
+                }
+
+                leaf monState {
+                    description "Configuration to set fabric link monitoring state: enable/disable";
+                    type string {
```

Comment:
if the acceptable values are `enable/disable` can we make the type as enum or pattern?

Reply:
If only accept enable/disable, we can remove length?

Reply:
And can be used by type stypes:mode-status;
Check example here: https://github.com/sonic-net/sonic-buildimage/blob/cab94964d94381ac399361436e85d6ca6637cbd9/src/sonic-yang-models/yang-models/sonic-interface.yang#L80